core/sandbox/.toolkitstate/local.json

```diff
@@ -0,0 +1,3 @@
+{
+  "port": 3001
```

Comment:
**issue**: I don't think this file should be tracked by git; this file is updated by the Node plugin on each run with whatever port it's chosen